README.md

```diff
@@ -100,136 +160,80 @@ This will take approximately a minute to run, with the majority of the time spen
-
-========================================= Kurtosis Modules =========================================
-GUID                    LocalPortBindings
-eth-module_1638991377   1111/tcp -> 127.0.0.1:50841
-```
+And that's it! Anything doable on the testnet is now doable against your private Ethereum network running in your Kurtosis enclave.
 
-Copy the interactive REPL's GUID, and replace both `YOUR_ENCLAVE_ID_HERE` and `YOUR_REPL_GUID_HERE` in the below command with the appropriate values:
+To destroy the enclave we created and everything inside it, run:
 
 ```
-kurtosis repl install YOUR_ENCLAVE_ID_HERE YOUR_REPL_GUID_HERE ethers
+kurtosis enclave rm -f demo
 ```
 
-When the command finishes, you can now use it in your CLI! (You can execute the next command in the interactive REPL that should be still open in the previous tab)
+To destroy all enclaves, run:
 
-```javascript
-const ethers = require("ethers")
-```
-
-Now let's get a connection to the node with service ID `bootnode` by getting a [JsonRpcProvider](https://docs.ethers.io/v5/api/providers/jsonrpc-provider/). First we'll grab the `ServiceContext` (Kurtosis' representation of the node), use it to get the ports that the bootnode is listening on, and then construct an Ethers client connected to the node:
-
-```javascript
-// Grab the bootnode's service context
-bootnodeServiceId = executeEthModuleResultObj.bootnode_service_id
-bootnodeNodeObj = executeEthModuleResultObj.node_info[bootnodeServiceId]
-getBootnodeServiceCtxResult = await enclaveCtx.getServiceContext(bootnodeServiceId)
-bootnodeServiceCtx = getBootnodeServiceCtxResult.value
-
-// Get the IP & port of the bootnode, inside the enclave
-bootnodeRpcPortId = bootnodeNodeObj.rpc_port_id
-bootnodeRpcPort = bootnodeServiceCtx.getPrivatePorts().get(bootnodeRpcPortId)
-bootnodePrivateIp = bootnodeServiceCtx.getPrivateIPAddress()
-
-// Instantiate the Ethers client
-bootnodeRpcProvider = new ethers.providers.JsonRpcProvider(`http://${bootnodePrivateIp}:${bootnodeRpcPort.number}`)
 ```
-
-Notice how we used the `executeEthModuleResultObj` object containing details about the Ethereum network, which we got from executing the module at the very beginning.
-
-Finally, let's verify that our Ethereum network is producing blocks:
-
-```javascript
-blockNumber = await bootnodeRpcProvider.getBlockNumber()
-if (blockNumber > 0) { console.log("All is well!"); }
+kurtosis clean -a
 ```
 
-And that's it! Anything doable in Ethers is now doable against your private Ethereum network running in your Kurtosis enclave.
-
-To exit out of the REPL you can enter any of:
-
-* Ctrl-D
-* Ctrl-C, twice
-* `.exit`
-
-Your enclave will stay running until you command it to stop. Run the following to tell Kurtosis to tear down your enclave, replacing `YOUR_ENCLAVE_ID` with the ID of your running enclave:
-
-```
-kurtosis enclave rm -f YOUR_ENCLAVE_ID
-```
+and Kurtosis will tear down all enclaves and everything inside.
```

Comment:
You can remove this line (we already say 'destroy enclave and everything inside') above